domain_utils/domain_utils.py

```diff
@@ -178,29 +192,36 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True, ex
     `urllib.parse.urlparse
     <https://docs.python.org/3.8/library/urllib.parse.html>`_.
 
-    Empty scheme e.g. ``my.domain.cloudfront.net``
-    are assumed to be http schemes.
+    A url is parsed if it has a qualifying scheme. The qualifying schemes are
+    ``http``, ``https``, ``ws`` and ``wss``. Websocket schemes can be omitted using
+    the ``parse_ws`` parameter. Additionally, the ``scheme_default`` parameter
+    provides a scheme where the url doesn't contain one. The default is ``http``
+    and so urls without a scheme will, by default, be considered as http and therfore
+    parsed.
 
-    If a URL has a port but no scheme, urlparse determines the scheme to
-    be the hostname and we do not handle this special case. In this case,
-    the url will be treated as a non-http scheme and the return value will
-    be determined by the ``drop_non_http`` setting.
+    What is returned for unparsed urls is determined by the ``return_unparsed``
+    parameter.
 
     Parameters
     ----------
     url : string
         The URL to be parsed
+    return_unparsed : boolean, optional
+        Action to take if scheme is not parsed e.g. ``file:`` or ``about:blank``.
+        If ``False``, the result for non parsed urls will be an empty string
+        If ``True``, the result will be the original url, e.g.
+        ``about:blank`` -> ``about:blank`` even if ``scheme=False``.
+        See method description to understand whether a URL is parsed or not.
```

Comment:
Which method description?

Reply:
I just meant the main docsctring for this method so that parameter info doesn't get clogged up. not sure what the best way to explain refer this is.

Reply:
Ah I see. I'm not sure there's a better way, so it's fine to keep as-is.